### PR TITLE
Episode 9, Part 1

### PR DESCRIPTION
**Pull request by sockeqwe**

### CODE REVIEW

**Comment by sockeqwe** [comment]
stop! there is a rss feed issue!

**Comment by sockeqwe** [comment]
oops, false alarm!

I was using the wrong url (not raw github file url) to our rss feed. 


http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode9-part1/feed.rss


Everything is cool 😄 

Sorry

File: feed.rss [hunk 1]
```diff
@@ -590,5 +590,71 @@
...
+		<enclosure url="https://artemzin.com/static/thecontext/episodes/The.Context.episode.9.part1.mp3" length="65819851" type="audio/mpeg"/>
+		<atom:contributor>
+			<atom:name>Alexey Tsvetkov</atom:name>
+			<atom:uri>https://twitter.com/tsvtkv</atom:uri>
+		</atom:contributor>
+		<itunes:summary>
+			<![CDATA[
+			<h1>Episode 9, Part 1: Kotlin Gradle plugin, compilers and build systems with Alexey Tsvetkov</h1>
+
+			<p><strong>Important note for Pocket Casts app users:</strong>
```
Comment (artem-zinnatullin) [inline feed.rss]: 👍

File: feed.rss [hunk 2]
```diff
@@ -590,5 +590,71 @@
...
+
+			<p>See this tutorial: <a href="http://support.pocketcasts.com/article/streaming-episodes/">Pocket Casts: Streaming</a></p>
+
+			<h3><a href="https://github.com/artem-zinnatullin/TheContext-Podcast">How to listen &amp; subscribe</a></h3>
+
+			<ul>
+			<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/62">Discussion after the episode</a></li>
+			</ul>
+
+			<p>In this episode we have talked with Alexey Tsvetkov who works at the Kotlin team at jetbrains about Kotlin's gradle plugin, compilers and build systems in general.</p>
```
Comment (artem-zinnatullin) [inline feed.rss]: `about` -> `on`?

Comment (artem-zinnatullin) [inline feed.rss]: I'll fix that

File: show_notes/Episode_9_Part1.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+See this tutorial: [Pocket Casts: Streaming](http://support.pocketcasts.com/article/streaming-episodes/)
+
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/62)
+
+
+In this episode we have talked with Alexey Tsvetkov who works at the Kotlin team at jetbrains about Kotlin's gradle plugin, compilers and build systems in general.
```
Comment (artem-zinnatullin) [inline show_notes/Episode_9_Part1.md]: about -> on (will do)

Comment (sockeqwe) [inline show_notes/Episode_9_Part1.md]: oops,  no clue what I was thinking about ...
should I change this or what are you trying to say with "(will do)"?

**Comment by artem-zinnatullin** [comment]
Thanks a ton for editing and preparing show notes!

**Comment by sockeqwe** [comment]
+1 for CI feed validation!

File: show_notes/Episode_9_Part1.md (hunk 3 above)
Comment (artem-zinnatullin) [inline show_notes/Episode_9_Part1.md]: I dodid it!

Comment (sockeqwe) [inline show_notes/Episode_9_Part1.md]: 👍

**Pull request merged**